components/BlockLocationListItem.vue

```diff
@@ -0,0 +1,368 @@
+        SvgIconEmail: () =>
+            import(
+                "~/node_modules/ucla-library-design-tokens/assets/svgs/icon-email"
+            ),
+        SvgIconPhone: () =>
+            import(
+                "~/node_modules/ucla-library-design-tokens/assets/svgs/icon-phone"
+            ),
+        SvgMoleculePlaceholder: () =>
+            import(
+                "~/node_modules/ucla-library-design-tokens/assets/svgs/molecule-placeholder"
+            ),
+        IconWithLink
+    },
+    props: {
+        image: {
+            type: String,
+            default: "",
+        },
+        title: {
+            type: String,
+            default: "",
+        },
+        to: {
+            type: String,
+            default: "",
+        },
+        day: {
+            type: String,
+            default: "",
+        },
+        hour: {
+            type: String,
+            default: "",
+        },
+        address: {
+            type: String,
+            default: "",
+        },
+        addressLink: {
+            type: String,
+            default: "",
+        },
+        amenities: {
+            type: Array,
+            default: () => [],
+        },
+        reserveSeat: {
+            type: String,
+            default: "",
+        },
+        isUclaLibrary: {
+            type: Boolean,
+            default: true
+        }
+    },
+    computed: {
+        classes() {
+            return [
+                "block-location-list-item",
+                `color-${this.cardTheme}`,
+            ]
+        },
+        cardTheme() {
+            return this.isUclaLibrary ? "ucla" : "affiliate"
+        },
+        isExternalLink() {
+            return this.to.includes("http") ? true : false
+        },
+        parsedTarget() {
+            return this.isExternalLink ? "blank" : "_self"
+        }
+    },
+}
+</script>
+
+<style lang="scss" scoped>
+.block-location-list-item {
+    overflow: hidden;
+    border: 2px solid var(--color-theme);
+    border-radius: var(--rounded-slightly-all);
+
+    width: 100%;
+    max-width: 928px;
+    padding: 48px 64px;
+
+    transition-property: box-shadow;
+    @include animate-normal;
+
+    // Themes
+    &.color-ucla {
+        --color-theme: var(--color-visit-fushia-02);
+    }
+    &.color-affiliate {
+        --color-theme: var(--color-primary-blue-02);
+    }
+
+    .card-container {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        width: 100%;
+        max-width: 800px;
+
+        .image {
+            margin-right: var(--space-xl);
+            width: 352px;
+            height: 352px;
+            background: var(--gradient-01);
+        }
+
+        $positionKey: random(3);
+        $positions: left, center, right;
+        $nth: nth($positions, $positionKey);
+
+        .affiliate {
+            background: url(~/node_modules/ucla-library-design-tokens/assets/svgs/molecule-placeholder.svg?url),
+            var(--gradient-01);
```

Comment:
try creating two divs with different class names, one for the svg and one for the gradient. background-position-x should only shift the svg, otherwise we get a funky stripe (screenshot attached). for reference, see: https://codepen.io/axamei/pen/yLpdBGg
![Screen Shot 2022-04-26 at 4 50 45 PM](https://user-images.githubusercontent.com/44713143/165410919-9cc1e6c3-76c7-40d2-99ee-edf313c2adbd.png)